geode-docs/tools_modules/http_session_mgmt/sticky_vs_non-sticky_overview.html.md.erb

```diff
@@ -0,0 +1,70 @@
+---
+title:  Sticky vs Non-Stick Session overview
```

Comment:
I don't have a strong preference between "non-sticky" and "non-stick," but I do think that we ought to use one term consistently, both within this section/page and across other pages within the documentation.